machamp-core/src/main/kotlin/ru/telamon/machamp/AsyncTaskProcessor.kt

```diff
@@ -36,7 +29,7 @@ class AsyncTaskProcessor(
             taskHandlersMap[it.getType()] = it
         }
 
-        jobs = Array(threadsCount) { i ->
+        jobs = List(parallelismLevel) { i ->
```

Comment:
@persifal It's hard naming here, but parallelism level looks misleading: https://www.tutorialspoint.com/what-are-the-different-levels-of-parallelism. I see its usage in https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/concurrent/ForkJoinPool.html but it sounds like artefact from Russian engineers.

Reply:
Yeah, agree the name might be not clear. Could you please suggest more meaningful variant to replace?

Reply:
@persifal `threadsCount` is my best guess. Actually, this value not only about performance of this module, but about the load it generates to external services.

Reply:
In this code we launch some coroutines which, in fact, are not threads.

Reply:
> In this code we launch some coroutines which, in fact, are not threads.

Yes. But historically it was threads, before Kotlin development was started. And coroutine that loops infinitely and make pauses are like threads emulated with coroutines.